src/test/java/com/adyen/RecurringTest.java

```diff
@@ -148,4 +172,42 @@ public void testStoreToken101() throws IOException {
 		}
 	}
 
+    @Test
+    public void testScheduleAccountUpdater() throws Exception {
```

Comment:
Is this test actually running on mvn install ?

Reply:
yes